gandalf.js

```diff
@@ -1,128 +1,132 @@
 /**
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?